src/components/thumbnail/collapsible-document-section.tsx

```diff
@@ -82,10 +88,22 @@ export const CollapsibleDocumentsSection: React.FC<IProps> = observer(
       </div>
       { isOpen &&
         <div className="list">
-          { documentNames.length > 0
-            ? documentNames.map((docName, i) =>
-                <div key={i} style={{padding: "5px 10px"}}>{`DOCUMENT: ${docName}`}</div>)
-            : <div style={{padding: "5px 10px"}}>No Documents</div>
+
+          { documentKeys.length > 0
+            ? documentKeys.map((key, i) => {
+              const document = networkDocuments.getDocument(key);
+              if (!document) return <div>Documen Missing</div>;
```

Comment:
```suggestion
              if (!document) return <div>Document Missing</div>;
```

Reply:
Thanks!  Jim caught that too.  Should be fixed.

Reply:
Will resolve.